src/components/SharedComponents/index.js

```diff
@@ -5,6 +5,7 @@ export { default as EvidenceButton } from "./Buttons/EvidenceButton";
 export { default as DateDisplay } from "./DateDisplay";
 export { default as Divider } from "./Divider/Divider";
 export { default as InlineUser } from "./InlineUser/InlineUser";
+export { default as Map } from "./Map";
```

Comment:
I don't think that we will need this Map here, I'd rather suspect we will end up with different interfaces in all the instances we show a map. The designs for all the different screens where we will end up having a map are not final yet, so I would not add this here for now.